src/software/world/team_types.h

```diff
@@ -2,6 +2,11 @@
 
 #include "software/util/make_enum/make_enum.h"
 
+/**
+ * The possible team side of SSL robots
```

Comment:
"The possible team side**s**..." to keep it consistent with comment below.

Reply:
fixed